src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonValue.CreateOverloads.cs

```diff
@@ -223,6 +224,7 @@ public partial class JsonValue
         /// <param name="value">The underlying value of the new <see cref="JsonValue"/> instance.</param>
         /// <param name="options">Options to control the behavior.</param>
         /// <returns>The new instance of the <see cref="JsonValue"/> class that contains the specified value.</returns>
+        [return: NotNullIfNotNull(nameof(value))]
         public static JsonValue? Create(string? value, JsonNodeOptions? options = null) => value != null ? new JsonValueTrimmable<string>(value, JsonMetadataServices.StringConverter) : null;
```

Comment:
Is it correct that options is entirely ignored here @eiriktsarpalis?  Why do all of these Create methods take options if they're then ignored?

Reply:
I believe it's due to symmetry with all the other `JsonObject`/`JsonArray` create methods:

https://github.com/dotnet/runtime/blob/7aef48a531e814bb8e8e0f6e060b100b51b27c80/src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonObject.cs#L52-L65

`JsonValue` happens to not need it because it simply encapsulates arbitrary .NET values but it might do in the future. cc @steveharter